env.example

```diff
@@ -143,14 +143,18 @@ ETHERPAD_SKIN_VARIANTS="super-light-toolbar super-light-editor light-background
 # LDAP authentication (for more information see the Cyrus SASL saslauthd.conf man page)
 #
 
-# LDAP url for connection
-#LDAP_URL=ldaps://ldap.domain.com/
+# LDAP url:port for connection
+#LDAP_URL=ldaps://ldap.domain.com:636
```

Comment:
`ldaps:` can only be specified if `ldap_start_tls` is not. Otherwise, `ldap` must be used instead.

Reply:
Added L:147